bytecode/cfg.py

```diff
@@ -74,6 +74,65 @@ def add_block(self, instructions=None):
         self._add_block(block)
         return block
 
+    def compute_stacksize(self):
+
+        def walk(block, depth, maxdepth):
+
+            if block.seen or block.startdepth >= depth:
+                return maxdepth
+
+            block.seen = True
+            block.startdepth = depth
+
+            for instr in block:
+
+                if isinstance(instr, SetLineno):
+                    continue
+
+                effect = instr.stack_effect
+                depth += effect
```

Comment:
It doesn't look like effect is used later, so I suggest:

depth += instr.stack_effect